docs/sources/pip.md

```diff
@@ -0,0 +1,22 @@
+# Pip
+
+The pip source uses `pip` CLI commands to enumerate dependencies and properties. It is expected that `pip` is available in the `virtual_env_dir` specific directory before running `licensed`.
+
+Your repository root should also contain a `requirements.txt` file which contains all the packages and dependences that are needed. You can generate one with `pip` using the command:
+```
+pip freeze > requirements.txt
+```
+
+A `virtualenv` is assumed to be setup before running `licensed`. You can setup a `virtualenv` by running the command:
+```
+virtualenv <your_venv_dir>
+```
+_note_: `<your_venv_dir>` path should be relative to the repository root.
+
+#### virtual_env_dir
```

Comment:
nit: not blocking, but I still think this could be more clear that the `virtual_env_dir` setting is required and can be either a relative or absolute path, with the relative path being evaluated from the repository root.